zowe-rest-api-sample-kotlin-spring/src/test/kotlin/org/zowe/sample/kotlin/apiservice/ZoweRestApiSampleKotlinSpringApplicationTests.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * This program and the accompanying materials are made available and may be used, at your option, under either:
+ * * Eclipse Public License v2.0, available at https://www.eclipse.org/legal/epl-v20.html, OR
+ * * Apache License, version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.sample.kotlin.apiservice
+
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.context.SpringBootTest
+
+@SpringBootTest
+class ZoweRestApiSampleKotlinSpringApplicationTests {
+
+    @Test
+    fun contextLoads() {
```

Comment:
No assert, just test initialization of spring context. I think this is tested in all other, right?

Reply:
This should test whether the application is able to start. I think this is the default test (automatically generated) for Spring Boot. But you are right, it is already tested by other tests. I will delete it.

Reply:
Deleted but lost 2% of coverage :-D